dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/dao/SimpleMongoDBStorageStrategy.java

```diff
@@ -53,8 +54,11 @@
     public static final String OBJECT_TYPE_VALUE = XMLSchema.ANYURI.stringValue();
     public static final String CONTEXT = "context";
     public static final String PREDICATE = "predicate";
-    public static final String OBJECT = "object";
+    public static final String PREDICATE_HASH = "predicate_hash";
+    public static final String OBJECT = "object_original";
```

Comment:
hey i don't think we need to hash predicates and subjects - just objects.  objects are possibly literals which means they can have unspecified length (and in practice are likely to be very long -- sometimes people literally put books into comments which are object values).  theoretically predicates and subjects are URIs which means that to be valid they are limited in length.  no harm in doing it, it just adds a layer of indirection at query time.